lms/models/event.py

```diff
@@ -14,6 +14,7 @@ class Type(str, Enum):
         CONFIGURED_LAUNCH = "configured_launch"
         DEEP_LINKING = "deep_linking"
         AUDIT_TRAIL = "audit"
+        EDITED_ASSIGNMENT = "edited_assignment"
```

Comment:
New event type

Reply:
The rationale at the time was to make the event table as slim as possible. Is the complexity of splitting the type to another table worth the extra bytes in the rows? I'm not sure. There're 17M rows now on that table now.

I reckon the enum has a lot of value in the codebase, it doesn't add a "string typed" war t and you (can) get autocomplete for the values etc.

Orthogonal to that is the db constraint based on those values from the enum. It does add some complexity to add/remove it when new types I reckon we thought it was not worth it at the time.

Reply:
We might be able to make the table slim by using an `int` for the `type` column rather than a string? Then wrap the ints in a Python enum so that `models.EventType.EDITED_ASSIGNMENT = 5` etc.